code/modules/cargo/packs/costumes_toys.dm

```diff
@@ -293,3 +293,15 @@
 	for(var/i in 1 to 10)
 		cardpacktype = pick(subtypesof(/obj/item/cardpack))
 		new cardpacktype(C)
+
+/datum/supply_pack/costumes_toys/stickers
+	name = "Sticker Pack Crate"
+	desc = "This crate contains a random assortment of stickers."
+	cost = CARGO_CRATE_VALUE * 3
+	contains = list()
+
+/datum/supply_pack/costumes_toys/stickers/fill(obj/structure/closet/crate/C)
+	for(var/i in 1 to rand(1,2))
+		new /obj/item/storage/box/stickers(C)
+	if(prob(30)) // a pair of googly eyes because funny
+		new /obj/item/storage/box/stickers/googly(C)
```

Comment:
```suggestion
/datum/supply_pack/costumes_toys/stickers/fill(obj/structure/closet/crate/crate)
	for(var/i in 1 to rand(1,2))
		new /obj/item/storage/box/stickers(crate)
	if(prob(30)) // a pair of googly eyes because funny
		new /obj/item/storage/box/stickers/googly(crate)
```